Organization & Loops Practice

bfwetmore opened the pull request:
- Add Directories for each lesson.
- Add Sub directories for challenges and practices
- Modify app.js file to a for loop

bfwetmore committed:
Organization & Loops Practice update.

- Add Directories for each lesson.
- Add Sub directories for challenges and practices
- Modify app.js file to a for loop

do-work commented on Loops/Loops Practice/index.html:
```diff
@@ -0,0 +1,12 @@
+<main></main>
+<script src="js/app.js"></script>
+</body>
+</html>
```
I would start adding new lines at the end of file.

bfwetmore replied:
New commit pushed addressing this and directory comment above.

do-work reviewed (changes requested):
Keep in mind common naming conventions and note that each language can have different conventions.
A lot of times I will look at a specific language 'topic' on Github - https://github.com/topics/javascript
From there I will browse for some of the top projects and look at their structure.
One example is https://github.com/axios/axios
Also you can look for language specific 'style guides'. 
This may have more info than what your looking for right now but the takeaway to me is that the file and dir names should be lowercase.

bfwetmore committed:
Address Comments.

-All lowercase on directory files.
-Added additional line at the end of each file.

do-work commented on loops/Loops Practice/css/style.css:
```diff
@@ -41,4 +41,4 @@ h2 {
     line-height: 0.8;
-}
\ No newline at end of file
+}
```
For the directory we don't want to have spaces in it. So it should be
`loops/loopsPractice` or `loops/loops-practice` - I've seen examples of both.

bfwetmore replied:
Fixed in latest commit

bfwetmore committed:
Address Comments.

-All lowercase on directories and files.
